OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -303,6 +334,12 @@ function Set-TargetResource {
         -OctopusServerUrl $OctopusServerUrl `
         -ApiKey $ApiKey
 
+    # Check to see if spacename is emtpy
+    if ([string]::IsNullOrEmpty($SpaceName))
+    {
+        # Assign default space name
+        $SpaceName = "Default"
```

Comment:
This space may not actually exist.
And if it does exist, it may not be the default space.